test/server/cards/02_SHD/units/GentleGiant.spec.ts

```diff
@@ -0,0 +1,87 @@
+describe('Gentle Giant', function() {
+    integration(function(contextRef) {
+        describe('Gentle Giant\'s ability', function() {
+            beforeEach(function () {
+                contextRef.setupTest({
+                    phase: 'action',
+                    player1: {
+                        groundArena: [
+                            { card: 'gentle-giant', damage: 1 },
+                            { card: 'r2d2#ignoring-protocol', damage: 3 },
+                            { card: 'c3po#protocol-droid', damage: 1 }],
+                    },
+                    player2: {
+                        groundArena: [{ card: 'wampa', damage: 1 }]
+                    }
+                });
+            });
+
+            it('should heal a target with 1 damage to full', function () {
```

Comment:
This case isn't really needed at this point, we have a lot of tests already confirming that healing to full works. Can remove it just to keep this test suite simple and easy to read